languages/julia/exercises/concept/vehicle-purchase/.meta/design.md

```diff
@@ -0,0 +1,25 @@
+## Learning objectives
+
+- Know the three common conditional statements in Julia
+  - `if`-statement
+  - Ternary operator
+  - Short-circuit Evaluation
+- Know when to use which of these
+
+## Out of scope
+
+- Nested ternary operators or short-circuits
+
+## Concepts
+
+The Concepts this exercise unlocks are:
+
+- `conditionals`: Exercises that require conditionals as prereq should only depend on this concept. Since these are technically all interchangable, depending on one of these versions makes no sense. However, for the website concept view, separating these into three concepts may make sense.
+- `if-statements`
+- `ternary-operator`
+- `short-circuit-evaluation`
```

Comment:
@ErikSchierboom or @iHiD: What's the best way to represent this in the `config.json` and `concepts/` dir?

The `conditionals` concept should solely be used for `config.json` purposes, where as the three more specific concepts should be used for `concepts/` documents. Or should I just have one large concept for the latter, too?

Reply:
Now that I think of it, perhaps splitting the exercise in three might be a good idea.

Reply:
@ErikSchierboom Jeremy and I have discussed this via DM, no need for you to respond anymore :) 

What we'll be doing:

- Have `concepts/conditionals` link to the three "subconcepts"
- Have a separate doc for the three subconcepts
- For progression purposes, all exercises should depend on `conditionals`

Reply:
Aha. Is that the reason why this PR doesn't yet has the concept documents?

Reply:
Yes

Reply:
I'll add these in a follow-up to keep the review simpler